frontend/vite.config.js

```diff
@@ -6,5 +6,13 @@ export default {
   },
   plugins: [
     createVuePlugin()
-  ]
+  ],
+  proxy: {
+    '/api': {
+      target: 'http://localhost:3001',
+      changeOrigin: true,
+      rewrite: path => path.replace(/^\/api/, '')
```

Comment:
I'm not sure if we should have this, because if we are serving both frontend and backend from express, then we have to put the backend routes behind `/api` anyway. Otherwise, what happens when both frontend and backend have a path that is `/students` (which is already the case)?

Reply:
Thank you! I missed out that the rewrite would make "/api" → "/" and conflict with the frontend's paths. I've removed it.

Reply:
No worries, this also means that in the backend, we will need to make sure all the backend routes are fronted by `/api`.